spec/intl.html

```diff
@@ -1401,11 +1401,66 @@ <h1>
+          </p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendarmonthdaytoisoreferencedate" type="abstract operation">
+          <h1>
+            CalendarMonthDayToISOReferenceDate (
+              _calendar_: a String,
+              _fields_: an ordinary Object with one or more own data properties,
+              _overflow_: a String,
+            ): either a normal completion containing a Record with fields [[ISOMonth]], [[ISODay]], and [[ReferenceISOYear]], or an abrupt completion
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>
+              It performs implementation-defined processing to convert _fields_, that describes a calendar date without a year (i.e., month code and day pair, or equivalent) in the calendar represented by _calendar_, to a reference date in the ISO calendar.
+              It performs the overflow correction given by _overflow_ on the date described by _fields_ in order to arrive at a valid month code and day pair in _calendar_.
+              For *"reject"*, values that do not form a valid date cause an exception to be thrown, as described below.
+              For *"constrain"*, values that do not form a valid date are clamped to the correct range.
+              It then returns a Record with a reference ISO date that corresponds to that month code and day pair in a reference year as described below.
+            </dd>
+          </dl>
+          <p>
+            The operation throws a *TypeError* exception if the own data properties present on _fields_ are insufficient to resolve to a month code and day pair in _calendar_.
+            For example, in calendars that use eras, a *TypeError* is thrown if _fields_ does not have one of the following combinations of properties:
+          </p>
+          <ul>
+            <li>*"monthCode"*, *"day"*</li>
+            <li>*"year"*, *"month"*, *"day"*</li>
+            <li>*"era"*, *"eraYear"*, *"month"*, *"day"*</li>
+          </ul>
+          <p>
+            The operation throws a *RangeError* exception if the date described by _fields_ does not exist, i.e. cannot be resolved to a month code and day pair, and _overflow_ is *"reject"*.
+            For example, when _calendar_ is *"gregory"*, _fields_ values of `{ monthCode: "M01", day: "32" }` and `{ "year": 2001, "month": 2, "day": 29 }` would both cause a *RangeError* to be thrown.
+            In the latter case, even though 02-29 is a date in the Gregorian calendar, a month code cannot be determined from the nonexistent date 2001-02-29.
+          </p>
+          <p>
+            The [[ReferenceISOYear]] field of the returned Record must be deterministically chosen for any particular calendar date in a given _calendar_.
+            The reference ISO year is the latest ISO year that is equal to or less than 1972 (the first ISO leap year after the epoch) which contains a date corresponding to that calendar date.
```

Comment:
Here's some draft language for this section that addresses the case where the same month/day can occur multiple times in an ISO year:

```suggestion
            To choose the reference ISO date, implementations should pick the latest ISO date where that month and day exists and that is earlier than or equal to December 31, 1972 in the ISO calendar.
```

Reply:
I'm trying to avoid "implementations should do X" in favour of "the reference ISO date is Y" but I've done something similar to this.